units/provisioning_convert_host_to_new.go

```diff
@@ -19,7 +19,6 @@ import (
 
 const (
 	convertHostToNewProvisioningJobName = "convert-host-to-new-provisioning"
-	maxProvisioningConversionAttempts   = 15
```

Comment:
Rather than delete this, it's supposed to be used on [L65](https://github.com/evergreen-ci/evergreen/pull/5673/files#diff-1dff55e2600df65798f1b093187554eef68fb91209d4efbf17911b06537d69ceL65) - can we re-add this and use it on L65?

Reply:
Done